imap_processing/tests/ultra/unit/test_decom_apid_880.py

```diff
@@ -2,57 +2,58 @@
 import pytest
 
 from imap_processing import decom
+from imap_processing.ultra.l0.decom_ultra import decom_ultra_apids
+from imap_processing.ultra.l0.ultra_utils import UltraParams
 
 
 @pytest.fixture()
-def decom_test_data(ccsds_path, xtce_aux_path):
+def decom_test_data(ccsds_path, xtce_path):
     """Read test data from file"""
-    data_packet_list = decom.decom_packets(ccsds_path, xtce_aux_path)
+    data_packet_list = decom.decom_packets(ccsds_path, xtce_path)
     return data_packet_list
 
 
-def test_aux_length(decom_test_data):
-    """Test if total packets in data file is correct"""
-    total_packets = 23
-    assert len(decom_test_data) == total_packets
-
-
 def test_aux_enumerated(decom_test_data):
     """Test if enumerated values derived correctly"""
 
+    count = 0  # count number of packets with APID 880
+    total_packets = 23
+
     for packet in decom_test_data:
```

Comment:
Might be simpler to filter for the APIDs right away?
```suggestion
    apid_880 = filter(lambda packet: packet.header["PKT_APID"].derived_value == 880, decom_test_data)
    assert len(apid_880) == total_packets
    for packet in apid_880:
```